docs/rfcs/0000-template.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: page
+title: <your title>
+permalink: /docs/rfcs/<file-name>
+---
+
+* RFC PR: [<PR #>](https://github.com/mozilla-mobile/android-components/pull/<#>)
+* Start date: YYYY-MM-DD (day of posting)
+* End date: YYYY-MM-DD (initial deadline for feedback, subject to blocking changes required)
+* Stakeholders: <github-username>, <github-username>
+
+## Summary
+
+This section should include a brief description of the proposal.
+
+## Motivation
+
+This section should include reasoning about why the proposal is useful. Examples, specific scenarios, 
+open bugs, records of performance metrics, and other empirical data are beneficial to include here if available.
+
+## Guide-level explanation
+
+This section should include a high-level walkthrough of the steps required to implement the proposal.
+
+## Reference-level explanation
+
+This optional section should include a detailed walkthrough of technical steps or code changes that 
+will be required to implement the proposal. Code samples and prototypes are beneficial to include here.
+
+## Drawbacks
+
+This section should include any drawbacks to the proposal. This section can be considered optional, but is highly encouraged in order to facilitate healthy debate.
+
+## Rationale and alternatives
+
+This section should include any alternative proposals considered, as well as the rationale for why
+they were not selected for the proposal. This section can be considered optional, but is highly encouraged in order to facilitate healthy debate.
+
+## Prior art
```

Comment:
I think this and Artifacts can be combined in one section called `Resources and Docs`, wdyt?

Reply:
I think it seems useful to have separate sections for external resources and artifacts that are being added to the tree as part of the RFC , but I'm open to changing the name. "Prior art" is just borrowed from earlier RFC examples